salt/metalk8s/orchestrate/deploy_node.sls

```diff
@@ -20,6 +20,26 @@ Install python36:
     - raw_shell: true
     - roster: kubernetes
 
+Set grains ssh:
+  salt.state:
+    - ssh: true
+    - roster: kubernetes
+    - tgt: {{ node_name }}
+    - saltenv: metalk8s-{{ version }}
+    - sls:
+      - metalk8s.node.grains
+    - require:
+      - metalk8s: Install python36
+
+Refresh grains:
+  metalk8s.saltutil_cmd:
+    - name: saltutil.sync_grains
+    - tgt: {{ node_name }}
+    - ssh: true
+    - roster: kubernetes
+    - require:
+      - salt: Set grains ssh
```

Comment:
Normally it's used to sync module yes but it also refreshes the grains and btw I don't really know why but just calling a `refresh_grains` does not work in salt-ssh here.

Btw we need to do it here otherwise we do not see the metalk8s grains in salt-ssh.
(Also in MetalK8s we do not have any custom grain module)